build.gradle

```diff
@@ -132,11 +132,15 @@ check.dependsOn 'spotlessApply'
 def Task getDockerBuildTask(String artifactName, String projectDir) {
     return task ("buildDockerImage-$artifactName"(type: DockerBuildImage) {
         def buildTag = System.getenv('VERSION') ?: 'dev'
-        def buildPlatform = System.getenv('DOCKER_BUILD_PLATFORM') ?: 'linux/amd64'
-        def alpineImage = System.getenv('ALPINE_IMAGE') ?: 'alpine:3.4'
-        def postgresImage = System.getenv('POSTGRES_IMAGE') ?: 'postgres:13-alpine'
         def jdkVersion = System.getenv('JDK_VERSION') ?: '17.0.1'
-        def buildArch = System.getenv('DOCKER_BUILD_ARCH') ?: 'amd64'
+
+        def arch = System.getProperty("os.arch").toLowerCase()
+        def isArm64 = arch == "aarch64" || arch == "arm64"
+
+        def buildPlatform = System.getenv('DOCKER_BUILD_PLATFORM') ?: isArm64 ? 'linux/arm64' : 'linux/amd64'
+        def alpineImage = System.getenv('ALPINE_IMAGE') ?: isArm64 ? 'arm64v8/alpine:3.14' : 'alpine:3.14'
+        def postgresImage = System.getenv('POSTGRES_IMAGE') ?: isArm64 ? 'arm64v8/postgres:13-alpine' : 'postgres:13-alpine'
```

Comment:
similar to above, there is [amd64/postgres:13-alpine](https://hub.docker.com/r/amd64/postgres)